### ISSUE TITLE
nidm to nimare dataset

### ISSUE THREAD

**Issue by Remi-Gau** [neurostuff/NiMARE#837]
sort of planning to write some small utility to create Nimare dataset from nidm results

I guess a good starting point would be this one: https://github.com/neurostuff/NiMARE/blob/63f0a7e19ec1e85d3e36ec7e24f0a123765df68a/examples/misc-notebooks/save_nidm_to_dset.ipynb

And then make it more general?

Is the plan to still have some nimare.io.nidm2nimare?

Anything else I should be aware of?

**Comment by Remi-Gau** [comment]
While having a look at this, I realize that this notebook seems obsolete, no?

https://github.com/neurostuff/NiMARE/blob/main/examples/misc-notebooks/nidm_pain_meta-analyses.ipynb

At least several imports fail.

Also feels like it has been replaced in part by some current examples in the gallery.

**Comment by jdkent** [comment]
Hi @Remi-Gau,

> While having a look at this, I realize that this notebook seems obsolete, no?

Yes, that notebook is obsolete, it's now more of an artifact of how the test_nidm_pain.json was created.

> sort of planning to write some small utility to create Nimare dataset from nidm results
> Is the plan to still have some nimare.io.nidm2nimare?

I am not currently working on this, but I am working on downloading information from neurovault into a NiMARE dataset, which would include nidm results.

**Comment by Remi-Gau** [comment]
Our usecase is more to generate nidm file "locally" and then use those. So would it help you if you also had a separate function to do the nidm to dataset to decouple it from the neurovault usecase? 

Context: this is some sort of internal mega-analysis, so eventhough things will get pushed to neurovault eventually, it is faster to iterate without the upload / download nidm results to neurovault (esp that the upload feature of nidm results does not seem work atm).

**Comment by jdkent** [comment]
yes, nidm is a separate concept from neurovault so having a function specifically for nidm would be useful!
